### PR TITLE
GYR admin should have access to all clients in GYR

### PR DESCRIPTION
**Pull request by sartan**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -25,6 +25,6 @@ def update
   private
 
   def user_params
-    params.require(:user).permit(:is_beta_tester, :vita_partner_id, :timezone)
+    params.require(:user).permit(*(:is_super_admin if current_user.is_super_admin?), :is_beta_tester, :vita_partner_id, :timezone)
```
Comment (sartan) [inline app/controllers/users_controller.rb:28]: `(*(:is_super_admin if current_user.is_super_admin?)...`

Needlessly clever? Probably.

Comment (jenny-heath) [reply app/controllers/users_controller.rb:28]: never seen the syntax but assuming that it doesn't pass in any argument if `current_user.is_super_admin?` is false?

**Comment by bytheway875** [comment]
in tax/returns edit, we should make sure to get the collection of assignable users from the intake's vita_partner instead of the admin's vita partner so that a superadmin can still assign!!

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (sartan) [reply app/controllers/users_controller.rb]:
```
puts(*(:is_super_admin if true), :is_beta_tester, :vita_partner_id, :timezone)
irb(main):001:0> puts(*(:is_super_admin if true), :is_beta_tester, :vita_partner_id, :timezone)
is_super_admin
is_beta_tester
vita_partner_id
timezone
=> nil


irb(main):003:0> puts(*(:is_super_admin if false), :is_beta_tester, :vita_partner_id, :timezone)
is_beta_tester
vita_partner_id
timezone
=> nil


irb(main):004:0> puts(*(:is_super_admin if nil), :is_beta_tester, :vita_partner_id, :timezone)
is_beta_tester
vita_partner_id
timezone
=> nil
```

**Pull request merged**